.github/workflows/publish-model-2-packages.yml

```diff
@@ -0,0 +1,21 @@
+name: Publish org.lflang to GitHub Packages
+on:
+  workflow_dispatch:
+  release:
+    types: [created]
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      - uses: actions/checkout@v3
+      - name: Prepare build environment
+        uses: ./.github/actions/prepare-build-env
+      - name: Publish package
+        uses: gradle/gradle-build-action@0d13054264b0bb894ded474f08ebb30921341cee
```

Comment:
Why use this specific hash? Maybe just use `v2.1.7`?

Reply:
In my excitement I just put what the GH actions gradle tutorial gave me. I fully agree with your suggestion.